sklearn/utils/estimator_checks.py

```diff
@@ -498,12 +502,12 @@ def _should_be_skipped_or_marked(estimator, check):
     return False, "placeholder reason that will never be used"
 
 
-def parametrize_with_checks(estimators):
+def parametrize_with_checks(estimators, check_yielder=None):
```

Comment:
I will revert this change because it don't need it anymore in the latest version of this PR. So it's probably a YAGNI.